docs/03_server/11_integration/01_rest-endpoints/03_basics.mdx

```diff
@@ -79,232 +79,498 @@ Data Servers are the most complex of the REST Endpoints provided by the Genesis
 
 For all Data Server requests, a `SOURCE_REF` header is mandatory, as it is used to match requests to the correct client subscription. This in turn allows for multiple subscriptions to the same query for the same session (think multiple grids in the UI using the same query with different filters).
 
-There follows a list of Data Server messages mapped to HTTP endpoints.
+### Endpoint
+The naming approach for the dataserver endpoint follows the lowercase hyphen-separated web standards convention (i.e. ALL_TRADES becomes all-trades). There is no need to rename your dataserver resource, the endpoint will be exposed using the correct naming convention automatically.
 
-### DATA_LOGON
-
-To initiate a Data Server query, a DATA_LOGON message is required to create the subscription. This is requested via a POST request to
+A sample endpoint url would look like this:
 `[host]:[genesis_router_port]/[data_server_query_name]`.
```

Comment:
we repeat this URL over and over for every single dataserver REST endpoint description. I have created a new Endpoint heading at the beginning to explain what the URL should look like.

Reply:
Good idea